lightning-invoice/src/payment.rs

```diff
@@ -15,9 +15,9 @@
 //! and payee using information provided by the payer and from the payee's [`Invoice`], when
 //! applicable.
 //!
-//! [`InvoicePayer`] is parameterized by a [`LockableScore`], which it uses for scoring failed and
-//! successful payment paths upon receiving [`Event::PaymentPathFailed`] and
-//! [`Event::PaymentPathSuccessful`] events, respectively.
+//! [`InvoicePayer`] uses its [`Router`] parameterization for scoring failed and successful payment
+//! paths upon receiving [`Event::PaymentPathFailed`] and [`Event::PaymentPathSuccessful`] events,
```

Comment:
Maybe update to mention the probing events also while you're here?

Reply:
Fixed in ea84acfe